chromium_src/chrome/browser/ui/views/side_panel/lens/lens_core_tab_side_panel_helper.cc

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "build/build_config.h"
+
+#if !BUILDFLAG(IS_ANDROID)
+#include "chrome/browser/ui/views/frame/browser_view.h"
+#endif
```

Comment:
files in `views` are only for desktop. we don't need `IS_ANDROID` check here

Reply:
Hmm, are we sure about that? CI complained and I see upstream using the same buildflag in that file here: https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/views/side_panel/lens/lens_core_tab_side_panel_helper.cc;l=16-23?q=lens_core_tab_side&ss=chromium

Reply:
oh, it's strange why upstream included this file only in `ui` target globally..

Reply:
Yup, we need this guard now as this file is inlucded in android also.
I think upstream file needs some cleanup (ex, some views only header files are included in android..)